app/src/main/java/io/gnosis/safe/ui/safe/add/AddSafeFragment.kt

```diff
@@ -72,6 +73,7 @@ class AddSafeFragment : BaseViewBindingFragment<FragmentAddSafeBinding>() {
         with(binding) {
             progress.visible(false)
             nextButton.isEnabled = false
+            bottomLabels.isVisible = false
```

Comment:
better to be consistent and use either isVisible or .visible(false) everywhere. We already use .visible(boolean) here, so I would continue using this extension